src/tests/Myr.test.js

```diff
@@ -138,12 +138,26 @@ describe("Updates to Myr's Model", () => {
         expect(colorRegEx.test(color)).toBeTruthy();
     });
 
+    it("to set Random Color", ()=> {
+        myr.cursor.color = null;
+        myr.setRandomColor();
+        expect(myr.cursor.color).not.toBeNull(); 
+    });
+
     it("pick Random Color out of a list of colors", () => {
```

Comment:
The tests for `getRandomColor` should check to make sure that the color attribute of the cursor object does not update